src/PresentationalComponents/RenderForms.js

```diff
@@ -1,9 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import FormTemplate from '@data-driven-forms/pf4-component-mapper/dist/cjs/form-template';
+import componentMapper from '@data-driven-forms/pf4-component-mapper/dist/cjs/component-mapper';
 import { Skeleton } from '@redhat-cloud-services/frontend-components';
 import { Stack, StackItem, Card, CardBody } from '@patternfly/react-core';
-import FormRender from '@data-driven-forms/react-form-renderer';
+
+import FormRender from '@data-driven-forms/react-form-renderer/dist/cjs/form-renderer';
+import componentTypes from '@data-driven-forms/react-form-renderer/dist/cjs/component-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/dist/cjs/validator-types';
+import validatorMapper from '@data-driven-forms/react-form-renderer/dist/cjs/validator-mapper';
+
+const componentMapperExtended = {
+    ...componentMapper,
```

Comment:
I think we don't have to include all components. Wizards, tabs, time picker, etc. can be committed atm since nobody is using them. We can always add it later

Reply:
But as the data are fetched from the API, you cannot be sure what components you really need.

Reply:
I am confident that the wizard and tabs are not used anywhere. But sure. We can keep it.

Reply:
tabs are used in cost-management.